eclair-core/src/main/scala/fr/acinq/eclair/payment/send/MultiPartPaymentLifecycle.scala

```diff
@@ -60,6 +54,7 @@ class MultiPartPaymentLifecycle(nodeParams: NodeParams, cfg: SendPaymentConfig,
   val id = cfg.id
   val paymentHash = cfg.paymentHash
   val start = System.currentTimeMillis
+  var retriedFailedChannels = false
```

Comment:
```suggestion
  private var retriedFailedChannels = false
```

Reply:
Done in https://github.com/ACINQ/eclair/pull/1439/commits/da4f8694edbfc83cfcb3e369079930aac028c748